drivers/flash/Kconfig.simulator

```diff
@@ -0,0 +1,158 @@
+# Kconfig - Flash simulator config
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig FLASH_SIMULATOR
+	bool
+	prompt "Flash simulator"
+	default n
+	depends on FLASH
```

Comment:
add dependency on MULTITHREADIN as k_wait and other system resources were used.

Reply:
All right!